### ISSUE TITLE
failed to convert svg to png with ImageMagick in pandoc export

### ISSUE THREAD

**Issue by czhiling** [shd101wyy/markdown-preview-enhanced#951]
Thanks a lot for developing great markdown-preview-enhanced.

I am using atom + mpe on macos mojave. My markdown includes some blocks of plantUML and in Chinese. I got following error messages in the pdf files from pandoc export.

```
Error: ImageMagick is required to be installed to convert svg to png.
Error: Command failed: magick /var/folders/2c/_0_4hpjd32l7ln81scpzk3hc0000gp/T/mume-svg11872
magick: xmlParseCharRef: invalid xmlChar value 8
 `No such file or directory` @ error/svg.c/SVGError/2980.
```

ImageMagick and pandoc has been install via homebrew.

Any idea how to solve this problem?

**Comment by czhiling** [comment]
No, I am still stuck here... I have tried many ways but without any luck.

**Comment by shd101wyy** [comment]
Hmm this is embarrassing. Which OS are you guys using. I tried on my Ubuntu and it worked fine.

**Comment by chiarld** [comment]
I am using macOS Mojave 10.14.

**Comment by czhiling** [comment]
Me too,macOS mojave

Danilo Chiarlone <notifications@github.com> 于2018年11月10日周六 上午7:53写道：

**Comment by kiyoshih** [comment]
I'm getting the same error too.

OS : Windows 10 Pro

**Comment by kiyoshih** [comment]
I may have found the cause.

- [mume/magick.ts](https://github.com/shd101wyy/mume/blob/ab159f944dbd9d2fec03a95ac311792de513a5e0/src/magick.ts#L13)
`convert` is no longer used.
- [[Resolved] convert.exe missing from 7.0.1-Q16?](http://www.imagemagick.org/discourse-server/viewtopic.php?t=29582)

**Comment by shumchi** [comment]
I met the same problem, my OS is MacOS 10.15.5.

I use the VS Code and install ImageMagic 7.0.10-19.

The problem was solved after setting the `Markdown-preview-enhanced: Image Magick Path` to `/usr/local/bin/magick` rather than empty.
